content/en/docs/event-sources/kernel/_index.md

```diff
@@ -12,8 +12,8 @@ Falco uses different instrumentations to analyze the system workload and pass se
 There are several supported drivers:
 
 - Kernel module *(default)*
-- Classic eBPF probe
 - Modern eBPF probe
+- Classic eBPF probe
```

Comment:
`Classic` or didn't we agree on `Legacy`? My preference would be `Legacy` as I believe most other docs call it that way.